### DISCUSSION TITLE
我的眼睛壓力山大

### DISCUSSION THREAD

**Discussion by HsuSungTing** [1101-GS4719-C/Python-Community-GS4719-C#1208] [General]
隨著現代科技的不斷發展和生活方式的改變，我們的生活當中無論是工作或娛樂，都很難離開各種螢幕，以筆者我而言，有高度近視(我父母都高度近視QQ...)，最近去眼科檢查，醫生叫我要吃葉黃素，而且每半年到一年要進行視網膜檢查(被各種顏色的探照燈照，然後給醫師看眼球狀況)，已下市推薦的是裡保健方法:

視力檢查：視力檢查是保持眼睛健康的關鍵。每年都要進行一次視力檢查，以確保眼睛沒有任何問題。如果您的眼睛有任何問題，請立即向醫生尋求幫助。

遠離屏幕：現代科技使我們更加依賴電腦、平板電腦和手機等電子產品，但長時間使用這些設備對眼睛有害。為了保持眼睛的健康，建議每小時休息5-10分鐘，並保持距離屏幕的適當距離，以減少眼睛的疲勞和壓力。

飲食健康：飲食對眼睛的健康非常重要。食物中的營養素，如維生素A、C和E、歐米加-3脂肪酸和鋅等，對保持眼睛的健康至關重要。飲食中應避免過多攝取糖分和飽和脂肪，這些食物會增加眼睛疾病的風險。

戒菸限酒：吸煙和飲酒都會對眼睛的健康造成負面影響。吸煙會增加患上白內障和視網膜病變的風險，而飲酒會對眼睛造成壓力和疲勞。

戶外活動：室內生活和工作容易讓眼睛疲勞和幹燥。建議在戶外活動和運動，以促進眼睛血液循環和新陳代謝。此外，陽光中的紫外線對眼睛有害，請戴上太陽眼鏡以保護眼睛。

小時候常常被媽媽嚇說，如果一直用電腦，眼睛會瞎掉......，長大後，用電腦和工作與唸書產生緊密關聯，不用電腦反而是一種解脫，不過我真的找不到什麼除了體力活之外的工作可以不用使用電腦的，目前期許自己，在合理的限度好好愛護眼睛，但也不要因此給自己太多限制。

**Comment by KevinCheng42** [comment]
我覺得戶外活動真的有助於減緩近視的度數，小時候父母或師長常會要求我們使用3C產品一段時間後要到戶外走走。根據衛服部的新聞，戶外活動的保護作用可能來自於陽光會增加視網膜多巴胺分泌的量，進而抑制眼軸伸長，抑或是待在戶外時，視野會更遠更廣，可以減少眼睛肌肉緊張，延緩眼球增長。

**Comment by zilinng** [comment]
我覺得現在要擁有健康的眼睛真的太難了，畢竟上課寫作業都需要使用平板或電腦，而我自己平常的娛樂最主要是追劇玩遊戲等需要3C產品的活動，等於做正事跟休息的時候都需要一直注視螢幕，對眼睛造成非常大的負擔，筆者提供的保護眼睛的方法讓我受益良多，目前我也在逼迫自己用戶外活動充實原本一直使用手機的空閒時間，希望可以藉此減輕對眼睛造成的傷害。

**Reply by lyien1021** [reply]
對啊尤其有些人一放假就一直追劇，甚至追到與世隔絕，這樣真的很傷眼睛！

**Reply by zilinng** [reply]
> 對啊尤其有些人一放假就一直追劇，甚至追到與世隔絕，這樣真的很傷眼睛！

怎麼會有人這樣！誰ㄚ！壞習慣得改！

**Comment by shuxian12** [comment]
我覺得要遠離螢幕真的是非常困難，尤其是對資工的學生，幾乎每門課都跟程式有關，不然就是要用平板
所以最後乾脆直接在宿舍接一台外接螢幕，不然真的會看到眼睛累爆ＱＱ 😢

**Comment by brian431** [comment]
生活在現代的人多多少少都會接觸到螢幕，因此視力問題是非常需要被重視的。尤其隨著年紀增長視力的問題也會逐漸浮現，定期做健康檢查並且隨時注意用眼狀況真的很重要。

**Comment by sjh501** [comment]
在現在不接觸到螢幕實在是太困難了，自身的視力也不好，一直都會有眼睛痠痛和眼壓高的問題。目前除了定時去眼科檢查，也開始改變自己使用螢幕的習慣：拉遠距離，縮短連續的使用時間。最近也開始尋找有護眼認證的電腦螢幕，希望這樣能夠對自身的眼睛好一點。

**Comment by Jack-1D** [comment]
感覺我現在就已經瘋狂用眼了。對我來說，出社會後好像只會更糟，可能連運動的時間都沒有，放假只想在家耍廢追劇。我認為應該要把出門走走當作一種例行公事，強迫自己的眼睛休息，才不會給自己找藉口。

**Comment by curtis1017** [comment]
眼睛是我們的靈魂之窗，他對於我們的身體有不可或缺的重要性，就算我們再怎麼樣的使用他，依然要留給眼睛休息的時間。那這樣的話，我有一個想法，那就是上課時間應該要改成30分鐘一堂課，這樣我們才有足夠的時間給眼睛休息。

**Comment by orianna901001** [comment]
之前遠距的時候，發現自己沒有3c產品真的不行，不僅上課用3c，休息用3c，根本沒有其他休閒娛樂能做，因為連書都是電子書QQ，只好天天來熱敷眼睛。

**Comment by llh26** [comment]
從小到大，父親都告訴我絕對不要在暗處用3C產品，也不要用不良的姿勢，例如：趴著、躺著⋯等，這些行為都會讓眼睛的負擔變大，而不聽話的後果就是視力越來越差、眼睛容易眼壓過高，更可能造成頭暈目眩，我自己就是很常因為這樣頭暈⋯，甚至頭痛到完全無法入睡，大家真的要好好保護自己的眼睛。

**Comment by ashleyy-huang** [comment]
從國小開始我就因為貪玩電腦所以近視要一直戴著眼鏡，隨著時間拉長，越來越體會到視力不好的不便，一拿掉眼鏡或隱眼就什麼都看不到，尤其是上大學開始習慣用平板作筆記增加了我看著螢幕的時間，每次去視力檢查都很害怕醫生說我的度數又加深了，最近正在努力提醒自己一段時間就要讓眼睛休息，看看外面的習慣，還有每天吃葉黃素，希望我的度數不會再加深:(

**Comment by CCello** [comment]
非常贊同你的觀點，現代科技的進步確實讓我們難以避免長時間使用各種螢幕，但是保持眼睛的健康也非常重要。你所列出的保健方法都是非常好的建議，例如每年定期進行視力檢查、遠離屏幕、飲食健康、戒菸限酒和戶外活動等等。
此外，還有一些其他的保健方法也值得考慮，例如經常閱讀紙質書籍、控制室內照明亮度、定期做眼保健操、使用抗藍光眼鏡等等。這些方法也都能夠幫助減輕眼睛的疲勞和保護眼睛的健康。
最後，雖然長時間使用螢幕確實會對眼睛造成壓力和疲勞，但是我們也不需要因此過度限制自己的生活。只要在使用螢幕時注意適當休息和保護眼睛的健康。

**Comment by alicia0725** [comment]
在高中以前沒有近視的我曾經完全不理解，但進到大學後天天不是對著電腦就是平板，近視度數急遽加深，我也發現常看3C產品一整天，會有眼壓過高導致頭痛想吐的問題，除了要時常多看外界事物外，我發現帶蒸氣眼罩對眼壓高非常有幫助！推薦給也有這個問題的人！

**Comment by tower20021202** [comment]
現在在上課時，我們不是在看著螢幕就是在看平板，傳統的筆記模式現在也都移到平板上了，真的對眼睛的負荷很大，因此上大學我的度數也增加了不少，此外，我也推薦常常用眼過度的人可以吃也黃素來保護我們的眼睛

**Reply by zephyrxin01** [reply]
我也有想過要買葉黃素來吃，但一罐好貴，而且我好像有看過一篇文章提到一般攝取均衡的飲食就可以提供一天所需的葉黃素，所以後來決定從多吃深綠色蔬菜開始。

**Comment by Jamesmoneya** [comment]
現在的學習模式根本離不開螢幕，早上起床開始從行事曆，關鬧鐘，回復訊息就開始使用螢幕到上課平板，回家用電腦打報告，其實眼睛根本完全沒有休息的時間，要遠離螢幕真的太難了，所以只好買一個超大的螢幕放在宿舍至少眼睛會舒服一點

**Comment by PaulSu0905** [comment]
我覺得適當調整屏幕亮度和對比度也很重要，確保屏幕的亮度和對比度設置在舒適的水平上，避免過亮或過暗的屏幕對眼睛的刺激。如果不強迫自己不用電子產品這會是很好的方法

**Comment by larry40305** [comment]
在工作和學習中無法完全避免使用電腦是現實的，但你已經意識到了保護眼睛健康的重要性。適當地關注並照顧眼睛，同時給自己一些適度的休息時間，是維護眼睛健康的關鍵。與此同時，也可以尋找其他興趣和活動，以平衡使用電腦的時間，並減少對眼睛的負擔。謹記注意眼睛健康，同時不要給自己太多限制，找到合適的平衡是關鍵。

**Comment by bestbanana295** [comment]
真的，好好保護自己的眼睛真的很重要，像是最近移植載盯著電腦螢幕感覺經都快要瞎了，感覺需要好好休息一夏，漸少眼睛的負擔才行，
大家真的要好好保重而且要好好注意身體健康畢竟身體好眼睛才會好，不知道大家有沒有甚麼對眼睛好的 良方呢？

**Comment by Miinteaa** [comment]
從小就因為遺傳問題近視戴著眼鏡，現在高度近視的我每天都要戴隱眼出門，有時候戴隱眼時間長了很容易造成眼睛乾澀，隱眼就開始跑片不舒服了🥲再者就是每每遇到期中考試週，睡眠過少也很容易造成眼睛乾澀，再戴上隱眼就雪上加霜了，但為了美觀我還是無法放棄隱形眼鏡😮‍💨

**Comment by zhiron544** [comment]
我覺得這篇文章很有幫助，讓我了解了一些保護眼睛的方法和原理。我也同意作者說的，現在要擁有健康的眼睛真的太難了，因為我也是一個常常使用電子產品的人。我想我應該多注意我的眼睛，並且多做一些戶外活動和飲食健康。謝謝作者分享這篇文章，希望大家都能保持眼睛的健康。
